modules/shadertools/src/lib/assemble-shaders.js

```diff
@@ -81,6 +81,8 @@ function assembleShader(
     glslVersion = 300; // TODO - regexp that matches atual version number
     versionLine = sourceLines[0];
     coreSource = sourceLines.slice(1).join('\n');
+  } else {
+    versionLine = `#version ${glslVersion}`;
```

Comment:
Nit: Good to add a comment

"// Ensure we always specify a version, e.g. Mac OS native bindings require this"

Reply:
This block seems redundant, given the update to `Shader`.